modules/tinymce/src/plugins/charmap/test/ts/browser/CharmapDialogHeightTest.ts

```diff
@@ -41,6 +40,11 @@ describe.skip('browser.tinymce.plugins.charmap.DialogHeightTest', () => {
       };
 
       it('TBA: Search for items, dialog height should not change when fewer items returned', async () => {
+        // Edge is being strange and focusing the body in this test but not when executed in isolation see #TINY-10579 for details
+        if (navigator.userAgent.includes(' Edg/')) {
+          return;
```

Comment:
Right thanks for pointing this out. It seems that we used the it vs it.skip method before so I might go with that. Would produce better logging so that we know that this was skipped so makes sense.